Simplified/Accounts/User/NYPLUserAccount.swift

```diff
@@ -453,13 +477,15 @@ private enum StorageKey: String {
   // MARK: - Remove
 
   func removeAll() {
+    // missed writing nil to _licensor????
```

Comment:
hmm looks like it... I'm wondering though is this was done elsewhere during the sign-out process, and if this was needed to not be nil in that process. Can you take a deeper look at that flow and verify if we are actually need to reset `licensor`?

Reply:
I'll see if this causes an issue. If it does, we can take care of this in a separate PR